In [30]:
from scipy.stats import norm
from math import sqrt

**Prob.27**. Confidence intervals Several factors are involved in the creation of a confidence interval. Among them are the sample size, the level of confidence, and the margin of error. Which statements are true?

a) For a given sample size, higher confidence means a smaller margin of error.

b) For a specified confidence level, larger samples provide smaller margins of error.

c) For a fixed margin of error, larger samples provide greater confidence.

d) For a given confidence level, halving the margin of error requires a sample twice as large.

**Answers to Prob. 27**

(a). False.  For a given sample size, higher confidence means a larger margin of error.

(b). True.  For a specified confidence level, larger samples provide smaller margins of error.

(c). False.  

(d). False.  

**Prob.28.** Confidence intervals, again Several factors are involved in the creation of a confidence interval. Among them are the sam- ple size, the level of confidence, and the margin of error. Which statements are true?

a) For a given sample size, reducing the margin of error will mean lower confidence.

b) For a certain confidence level, you can get a smaller margin of error by selecting a bigger sample.

c) For a fixed margin of error, smaller samples will mean lower confidence.

d) For a given confidence level, a sample 9 times as large will make a margin of error one third as big.

**Answers to Prob.28**

(a) True.  For a given sample size, reducing the margin of error will mean lower confidence.

(b) False.  For a certain confidence level, you cannot get a smaller margin of error by selecting a bigger sample.

(c) False.  

(d) False.

**Prob.38** Junk mail Direct mail advertisers send solicitations (a.k.a. “junk mail”) to thousands of potential customers in the hope that some will buy the company’s product. The acceptance rate is usually quite low. Suppose a company wants to test the response to a new flyer, and sends it to 1000 people randomly selected from their mailing list of over 200,000 people. They get orders from 123 of the recipients.

a) Create a 90% confidence interval for the percentage of people the company contacts who may buy something.

Solution: 

We know, in one random sample, $n=1,000$, and $k=123$, so $\hat{p}=0.123$.

90\% confidence level corresponds to a z value of $1.645$.

The confidence interval is $\hat{p} \pm z\cdot\sqrt{\frac{\hat{p}(1-\hat{p})}{n}}$=$(0.106, 0.140)$

b) Explain what this interval means.

The interval means that, at a confidence level of 90\%, the true proportion is encircled (emcompassed) by this interval.

c) Explain what “90% confidence” means.

It means that, if such confidence intervals are repeatedly constructed following the above procedure, 90\% of the time the constructed interval encompasses the true proportion.

d) The company must decide whether to now do a mass mailing.

The mailing won’t be cost-effective unless it produces at least a 5% return. What does your confidence interval suggest? Explain.

The above CI suggests that the true response rate is at least 5\% (with a confidence level at 90\%).  So the company may decide to mass mailing.

**Suppose the company now requires at least a 11\% response rate, how can we improve the study?**

The answer is to increase $n$.

Consider a situation that $n=10,000$ and $k=1,230$.  We reach the same $\hat{p}$.  z value remains the same.

The only thing affected in this situation is the standard error, $\sqrt{\frac{\hat{p}(1-\hat{p})}{n}} = 0.0054$.  The confidence interval becomes $(0.118, 0.128)$, and luckily the company may still go ahead with mass mailing.

In [13]:
def sol38a():
    z = norm.ppf(0.95)
    print(f'{z=}')
    phat = 123 / 1000
    n = 1000
    ME = z * sqrt(phat * (1-phat) / n)
    print(f'{ME=}')
    print(phat-ME)
    print(phat+ME)

sol38a()

z=1.6448536269514722
ME=0.017083608457721285
0.10591639154227872
0.14008360845772128


**Prob46**. Back to campus ACT, Inc. reported that 74% of 1644 randomly selected college freshmen returned to college the next year. The study was stratified by type of college—public or private. The retention rates were 71.9% among 505 students enrolled in public colleges and 74.9% among 1139 students enrolled in private colleges.

a) Will the 95% confidence interval for the true national retention rate in private colleges be wider or narrower than the 95% confidence interval for the retention rate in public colleges? Explain.

b) Do you expect the margin of error for the overall retention rate to be larger or smaller? Explain.

In [21]:
def sol46a1():
    z = norm.ppf(0.975)
    print(f'{z=}')
    phat = 0.749
    n = 1139
    ME = z * sqrt (phat * (1-phat) / n)
    print(f'{ME=}')
    print(phat-ME)
    print(phat+ME)

sol46a1()

z=1.959963984540054
ME=0.02518047965394772
0.7238195203460522
0.7741804796539478


In [22]:
def sol46a2():
    z = norm.ppf(0.975)
    print(f'{z=}')
    phat = 0.719
    n = 505
    ME = z * sqrt (phat * (1-phat) / n)
    print(f'{ME=}')
    print(phat-ME)
    print(phat+ME)

sol46a2()

z=1.959963984540054
ME=0.039203063597051904
0.679796936402948
0.7582030635970519


**Answer to Prob. 46** Both private and public of 95% confidence corresponse to a z value of $1.95$. The confidence interval of private colleges is $\hat{p} \pm z\cdot\sqrt{\frac{\hat{p}(1-\hat{p})}{n}}$=$(0.724, 0.774)$.  The confidence interval of public colleges is $\hat{p} \pm z\cdot\sqrt{\frac{\hat{p}(1-\hat{p})}{n}}$=$(0.680, 0.758)$.  Therefore, the 95% confidence interval for the true retention rate in private colleges will be wider than the 95% confidence interval for the retention rate in public colleges.

In [34]:
def sol46b():
    z = norm.ppf(0.975)
    print(f'{z=}')
    phat = 0.74
    n = 1644
    ME = z * sqrt (phat * (1-phat) / n)
    print(f'{ME=}')
    print(phat-ME)
    print(phat+ME)

sol46b()

z=1.959963984540054
ME=0.02120311741272906
0.7187968825872709
0.7612031174127291


**Answer to Prob.46(b)** I expect the margin of error for the overall retention rate to be smaller.